### PR TITLE
Ports Pulse Annihilator from the accursed lands

### PR DESCRIPTION
**Pull request by Buggy123**
The Pulse Annihilator, a scattershot version of the Pulse Destroyer. For when complete destruction isn't dead enough.

:cl: 
add: Having reviewed the latest advances in military technology, Nanotrasen has decided that the high-yield weaponry currently available to their elite policing forces is insufficient to combat the latest threats. They have tasked a subsidary, Paramilitary Advancement Research And Discovery Inc., Superweapon Equipment division, with developing a suitable upgrade.
/:cl:

### CODE REVIEW

**Comment by adamkad1** [comment]
Moar DESTROY

File: code/modules/projectiles/guns/energy/pulse.dm [hunk 1]
```diff
@@ -71,6 +71,14 @@
 /obj/item/gun/energy/pulse/destroyer/attack_self(mob/living/user)
 	to_chat(user, "<span class='danger'>[src.name] has three settings, and they are all DESTROY.</span>")
 
+/obj/item/gun/energy/pulse/destroyer/annihilator
+	name = "pulse ANNIHILATOR"
+	desc = "For when the situation calls for a little more than a pulse destroyer."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/scatter/pulse)
+
+/obj/item/weapon/gun/energy/pulse/destroyer/annihilator/attack_self(mob/living/user)
+	user << "<span class='danger'>[src.name] has no settings, only ANNIHILATION!</span>"
```
Comment (Couls) [inline code/modules/projectiles/guns/energy/pulse.dm]:
```suggestion
	to_chat(user, "<span class='danger'>[src.name] has no settings, only ANNIHILATION!</span>")
```
The grammar is also questionable but.. meh, only ANNIHILATE would make more sense

**Comment by LemonInTheDark** [comment]
Doesn't compile, I'm sure you can fix that easy.
What pr are we porting from, and who is the guy who made it?

**Comment by Ohagi-Chan** [comment]
P.A.R.A.D.I.S.E. hmmmmm?

**Comment by Fikou** [comment]
isnt this an easy varedit for admins

**Comment by optimumtact** [comment]
this is poorly ported and doesn't really fit any niche.

**Pull request closed without merging**